.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "\U0001F41B Bug Report"
+about: Something isn't working as expected
+---
```

Comment:
Actually we don't have a label for bug here. Should we add one?

Reply:
you can add one